kdc/krb5tgs.c

```diff
@@ -1620,7 +1604,144 @@ tgs_build_reply(astgs_request_t priv,
 	krb5_enctype etype;
 
 	if(b->kdc_options.enc_tkt_in_skey) {
+	    Ticket *t;
```

Comment:
You just moved this code around, rather than changing it, right?

Reply:
That was the intention, yes.